tests/testthat/test-on-failure.R

```diff
@@ -0,0 +1,12 @@
+context("on-failure")
+
+test_that("on_failure should work", {
+    is_red <- function(x){x == "red"}
+    on_failure(is_red) <- function(call, env){
```

Comment:
Looks like too space spaces in indent

Reply:
got it! Fixed the space `){` --> `) {` thing and changed indentation to two spaces for consistency with the rest of the files.